### DISCUSSION TITLE
After Prestige Template Deployment in Remote Server, CSS does not work properly.

### DISCUSSION THREAD

**Discussion by kalaylevent** [primefaces/community#1849] [PrimeNG]
I am working to deploy my angular prestige project in nginx server.  When i deploy it,  page seems messy without css settings etc However, when i control browser network, css files are loaded successfuly. How can i solve my problem.

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share the steps or any information to replicate and investigate the issue?

**Comment by kalaylevent** [comment]
Firstly, I build the project with ` npm build --configuration=production `
then  i work dockerfile by building and running my docker file : 
```
FROM nginx:alpine

# Nginx için çalışma dizinini ayarlayın
WORKDIR /usr/share/nginx/html
# Nginx yapılandırma dosyasını kopyalayı
RUN rm /etc/nginx/conf.d/default.conf
COPY default.conf /etc/nginx/conf.d/


# Önceden oluşturulan üretim dosyalarını Nginx'e kopyalayın
COPY /dist/prestige-ng .
# Port 8080'i dış dünyaya açın
EXPOSE 80

# Nginx'i başlatın ve daemon modunda çalıştırın
CMD ["nginx", "-g", "daemon off;"]     
```
my default.conf:
```
server {
    listen 80;

    server_name localhost;

    root /usr/share/nginx/html;
    index index.html;

    location / {
        try_files $uri $uri/ /index.html;
    }
}

```
then seems i go to url. css and js are loaded but it does not work correctly.

**Comment by kalaylevent** [comment]
I found the problem which is related with css files. Like theme-blue.css and layout-blue.css files loaded with failures in browser source tab. So, these files include html```
 <!DOCTYPE html><html lang="en"><head>
    <meta charset="utf-8">
    <title>Prestige</title>
    <base href="/">

    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="icon" type="image/x-icon" href="favicon.ico">
    <link id="theme-css" rel="stylesheet" type="text/css" href="assets/theme/theme-blue.css">
    <link id="layout-css" rel="stylesheet" type="text/css" href="assets/layout/css/layout-blue.css">
<link rel="stylesheet" href="styles.e47068f30dc3b5f6.css" media="print" onload="this.media='all'"><noscript><link rel="stylesheet" href="styles.e47068f30dc3b5f6.css"></noscript></head>
<body>
<app-root>
    <div class="loader-wrapper">
        <div class="loader"></div>
        <div class="loader-section section-left"></div>
        <div class="loader-section section-right"></div>
    </div>
</app-root>
<script src="runtime.2c9b4a6f065fc78d.js" type="module"></script><script src="polyfills.63398b7e76be1941.js" type="module"></script><script src="scripts.011fbf938f04b525.js" defer></script><script src="main.6ef6d40bcfafc7c1.js" type="module"></script>

</body></html>
```  instead of css rows
.

**Comment by kalaylevent** [comment]
problem solved. it is related with gitignore file including assets/css.
